{"name": "test_Android", "status": "broken", "statusDetails": {"message": "TypeError: 'str' object is not callable", "trace": "@pytest.mark.run(order=1)\n    @allure.severity(\"blocker\")\n    @allure.description(\"测试 小程序自动化疏通测试-Android\")\n    def test_Android():\n        startSmartXueWeiApp()\n        sleep(2)\n>       mobileDriver(text='同意并继续').click()\nE       TypeError: 'str' object is not callable\n\ntest_flow\\test_Authentication\\test_jenkinsDebug.py:45: TypeError"}, "description": "测试 小程序自动化疏通测试-Android", "attachments": [{"name": "log", "source": "4d650583-d2c5-4567-86f1-da6d5e1a2616-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "46d3f8d4-732a-4f4c-b9f5-9d1e0bce3e10-attachment.txt", "type": "text/plain"}], "start": 1637547882270, "stop": 1637547885307, "uuid": "c28a6f8e-515f-4d17-8bf7-93e45bb1ea52", "historyId": "037f6ac01dca35d949e1d56271286eaa", "testCaseId": "faad5a259c6669bfb35d617fe89efcc0", "fullName": "test_flow.test_Authentication.test_jenkinsDebug#test_Android", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_flow.test_Authentication"}, {"name": "suite", "value": "test_jenkinsDebug"}, {"name": "host", "value": "BF-202011271053"}, {"name": "thread", "value": "12532-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_flow.test_Authentication.test_jenkinsDebug"}]}